src/routes/NewReservationPage.js

```diff
@@ -0,0 +1,119 @@
+import {
+  React, useEffect, useRef, useState,
+} from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { getWorkspaces } from '../redux/workspaces/workspacesSlice';
+import { postReservation } from '../redux/reservations/reservationsSlice';
+
+const NewReservationPage = () => {
```

Comment:
- [ ] I was able to reserve in the past:

![image](https://github.com/Zilola-Nazarova/workspace-reservation-front-end/assets/57421823/4536e01f-9903-46b0-9103-9197a76fb9ee)

Kindly, add some validation so your users are only able to reserve in the future. 👍